CONTRIBUTING.md

```diff
@@ -45,8 +50,15 @@ Given that you have a compatible environment as stated above you can now set up
 #### Snapshot testing
 Create an AVD using the following command
 
+> Note: Currently, screenshot testing doesn't work properly on M1 chips.
```

Comment:
Maybe add something what to do in this case (eg generate screenshot tests on a different machine/check with another engineer if not available)

Reply:
This solution should come as an output of the ticket I created :)

Reply:
yeah but in the meantime we need have the right instructions - particularly with m1s rolled out more widely soon

Reply:
Done